Controller/ForgotResetController.php

```diff
@@ -52,6 +52,8 @@ public function __construct(
      */
     public function forgotPasswordAction(Request $request)
     {
+        $generic_msg = 'Your request has been sent.';
```

Comment:
Per Dayspring standard practice, properties should be `camelCase`.

Reply:
@jwong-dayspring Do you think we should also tack on the text: "Please check your email for further instructions.", or if it's more ideal to keep the generic msg as is?

Reply:
@ssitu-dayspring It's probably good to be helpful in the text. Something like "If an account was found, an email has been sent. Please check your email for further instructions."